submissions/kyle-earnest/scripts/article.js

```diff
@@ -44,6 +44,19 @@ Article.loadAll = function(dataWePassIn) {
  source, process it, then hand off control to the View. */
 Article.fetchAll = function() {
   if (localStorage.hackerIpsum) {
+    $.ajax({
+      url: 'data/hackerIpsum.json',
+      success: function (data, message, xhr) {
+        var eTag = xhr.getResponseHeader('eTag');
+        console.log(message, eTag);
+        localStorage.eTag = eTag;
+      }
+      if (localStorage.eTag === eTag){
```

Comment:
You closed your success function on the previous line. If you want all this stuff to run after you get the eTag, you'll have to put it all in the success function. Remember, the $.ajax takes a JS object (so you can't just put arbitrary code in there) and you never know when an AJAX call is going to come back.